recipe/meta.yaml

```diff
@@ -10,50 +10,122 @@ source:
   sha256: 0715d4a0393437aa2a48c144756360f9b5292635a05fd15c38cbbaf04ef5acb9
 
 build:
-  entry_points:
-    - connexion = connexion.cli:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
-requirements:
-  host:
-    - python {{ python_min }}
-    - poetry-core >=1.2.0
-    - pip
+outputs:
+  - name: connexion-base
+    build:
+      entry_points:
+        - connexion = connexion.cli:main
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
This unfortunately doesn't work for multiple outputs.  We have to define a separate `install_connection.sh` script that runs this same command.

Reply:
Are you saying this needs to be repeated for each output and are recommending a script to handle it?

Reply:
No, it's just that the script for this first output has to be an explicit file that's part of the recipe, not a command to run.  I don't know why this was the implementation but this command syntax just does't work with multiple outputs.  Instead, you need:
```
script: install_connexion.sh
```
And then in `recipe/`, you need to make a new file `install_connexion.sh`:
``` bash
#!/usr/bin/env bash

$PYTHON -m pip install . -vv --no-deps --no-build-isolation
```